tests/test_glue.py

```diff
@@ -216,6 +216,58 @@ def test_glue_job_untag(self):
         self.assertEqual(tags.get('Tags'), {})
         self.assertEqual(len(resources), 1)
 
+    def test_glue_crawler_tag(self):
+        session_factory = self.replay_flight_data("test_crawler_tags")
+        client = session_factory().client("glue")
+
+        tags = client.get_tags(ResourceArn='arn:aws:glue:us-east-1:644160558196:crawler/test')
```

Comment:
rather than getting the tags here you should just put this in the policy, add it to the filters block that no tags are on the resource